unlock-app/src/hooks/useCrossChainRoutes.ts

```diff
@@ -276,6 +276,14 @@ export const useCrossChainRoutes = ({
       (data?: CrossChainRouteWithBalance | null) => !!data
     ) as CrossChainRouteWithBalance[]
 
+  // Function to refetch all queries
+  const refetch = async () => {
```

Comment:
I don't thimk we need this anymore, do we?

Reply:
nope! cleaning up in a jiffy!